src/collect/collector.rs

```diff
@@ -151,6 +152,10 @@ impl Collectors {
             }
         }
 
+        // Initialize tracking & filters.
+        init_tracking(&mut self.probes)?;
```

Comment:
We're changing the default behavior here. Now we're attaching the skb-tracking-related probes.
I'm not saying it's necessarily bad, just pointing it out. Do you see any reason why this would not be wanted? do we need a flag?